src/runtime/hexagon_remote/sim_remote.cpp

```diff
@@ -234,13 +234,6 @@ void set_rpc_return(int value) {
 }
 
 int main(int argc, const char **argv) {
-    // The simulator needs this call to enable dlopen to work...
-    char libgcc[] = "libgcc.so";
-    char libc[] = "libc.so";
-    char libstdcpp[] = "libstdc++.so";
-    char *builtin[] = {libgcc, libc, libstdcpp};
-    dlinit(3, builtin);
```

Comment:
They were definitely needed at one point (I didn't like that that blurb was necessary), but something about the new linker command is making things work the way they were supposed to I guess.